howso/hierarchy.amlg

```diff
@@ -751,6 +751,177 @@
 		(call !Return (assoc payload (assoc "count" (size cases_to_move)) ))
 	)
 
+	;method to combine cases from all subtrainees into the root trainee and then deletes the subtrainees and removes them from the hierarchy
+	;{long_running (true) idempotent (true)}
```

Comment:
Should this be tagged `protected` as well? Seems like something that should not be available in the public platform API?